zeppelin-zengine/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -589,6 +598,7 @@ public String getWebsocketMaxTextMessageSize() {
     ZEPPELIN_INTERPRETER_JSON("zeppelin.interpreter.setting", "interpreter-setting.json"),
     ZEPPELIN_INTERPRETER_DIR("zeppelin.interpreter.dir", "interpreter"),
     ZEPPELIN_INTERPRETER_LOCALREPO("zeppelin.interpreter.localRepo", "local-repo"),
+    ZEPPELIN_INTERPRETER_REMOTEREPO("zeppelin.interpreter.remoteRepo", "remote-repo"),
```

Comment:
`ZEPPELIN_INTERPRETER_LOCALREPO` is local directory for both maven (for interpreter dependency) and npm (for helium).

So env variable name includes "maven", such as `ZEPPELIN_INTERPRETER_DEP_MVNREPO` could be less confusing, i think.